implementations/rust/ockam/ockam_node_test_attribute/tests/node/pass_when_used_on_any_fn.rs

```diff
@@ -0,0 +1,4 @@
+#[ockam_node_test_attribute::node]
+async fn foo(mut c: ockam::Context) {
```

Comment:
I think because macro will expand into main() function, it makes sense to me that function used with this macro must be called main. E.g. this won't compile, which is misleading
```rs
#[ockam_node_test_attribute::node]
async fn foo(ctx: &Context) {
    ctx.stop().await.unwrap();
}

fn main() {}
```rs
This doesn't compile, which is understandable
```
#[ockam_node_test_attribute::node]
async fn main(ctx: &Context) {
    ctx.stop().await.unwrap();
}

fn main() {}
```

Reply:
"must be called main" as we should force it in the macro? or just change the naming in the tests?

Reply:
OK, I got you. Makes total sense.